docs/ops.html

```diff
@@ -935,6 +974,236 @@
+      <td>kafka.consumer:type=consumer-fetch-manager-metrics,client-id=([-.\w]+)</td>
+    </tr>
+    <tr>
+      <td>fetch-latency-avg</td>
+      <td>The average time taken for a fetch request</td>
+      <td>kafka.consumer:type=consumer-fetch-manager-metrics,client-id=([-.\w]+)</td>
+    </tr>
+    <tr>
+      <td>fetch-latency-max</td>
+      <td>The max time taken for a fetch request</td>
+      <td>kafka.consumer:type=consumer-fetch-manager-metrics,client-id=([-.\w]+)</td>
+    </tr>
+    <tr>
+      <td>fetch-rate</td>
+      <td>The number of fetch requests per second</td>
+      <td>kafka.consumer:type=consumer-fetch-manager-metrics,client-id=([-.\w]+)</td>
+    </tr>
+    <tr>
+      <td>records-lag-max</td>
+      <td>The maximum lag in terms of number of records for any partition in this window</td>
+      <td>kafka.consumer:type=consumer-fetch-manager-metrics,client-id=([-.\w]+)</td>
+    </tr>
+    <tr>
+      <td>fetch-throttle-time-avg</td>
+      <td>The average throttle time in ms</td>
+      <td>kafka.consumer:type=consumer-fetch-manager-metrics,client-id=([-.\w]+)</td>
+    </tr>
+    <tr>
+      <td>fetch-throttle-time-max</td>
+      <td>The maximum throttle time in ms</td>
+      <td>kafka.consumer:type=consumer-fetch-manager-metrics,client-id=([-.\w]+)</td>
+    </tr>
+  </tbody>
+</table>
+
+
+<h5><a id="topic_fetch_monitoring" href="#topic_fetch_monitoring">Topic-level Fetch Metrics</a></h5>
+
+<table class="data-table">
+  <tbody>
+    <tr>
+      <th>Metric/Attribute name</th>
+      <th>Description</th>
+      <th>Mbean name</th>
+    </tr>
+    <tr>
+      <td>fetch-size-avg</td>
+      <td>The average number of bytes fetched per request for a specific topic.</td>
+      <td>kafka.consumer:type=consumer-fetch-manager-metrics,client-id=([-.\w]+),topic=([-.\w]+)</td>
+    </tr>
+    <tr>
+      <td>fetch-size-max</td>
+      <td>The maximum number of bytes fetched per request for a specific topic.</td>
+      <td>kafka.consumer:type=consumer-fetch-manager-metrics,client-id=([-.\w]+),topic=([-.\w]+)</td>
+    </tr>
+    <tr>
+      <td>bytes-consumed-rate</td>
+      <td>The average number of bytes consumed per second for a specific topic.</td>
+      <td>kafka.consumer:type=consumer-fetch-manager-metrics,client-id=([-.\w]+),topic=([-.\w]+)</td>
+    </tr>
+    <tr>
+      <td>records-per-request-avg</td>
+      <td>The average number of records in each request for a specific topic.</td>
+      <td>kafka.consumer:type=consumer-fetch-manager-metrics,client-id=([-.\w]+),topic=([-.\w]+)</td>
+    </tr>
+    <tr>
+      <td>records-consumed-rate</td>
+      <td>The average number of records consumed per second for a specific topic.</td>
+      <td>kafka.consumer:type=consumer-fetch-manager-metrics,client-id=([-.\w]+),topic=([-.\w]+)</td>
+    </tr>
+  </tbody>
+</table>
+
+<h5><a id="consumer_global_request_monitoring" href="#consumer_global_request_monitoring">Global Request Metrics</a></h5>
```

Comment:
I think all of these metrics come from `Selector`, so they should really be in the common section above. In fact, it looks like all of them except `request-latency-avg` and `request-latency-max` are there already. Maybe we can remove this section and add those two metrics to the common table?